src/modal.tsx

```diff
@@ -471,7 +571,7 @@ class ReactNativeModal extends Component {
           if (!this.props.isVisible) {
             this.close();
           } else {
-            this.props.onModalShow();
+            this.props.onModalShow!();
```

Comment:
Do we actually need the `!` here even if the prop has a default? (same in a few other cases below)

Reply:
I forgot to remove it, thanks !